.github/workflows/gource.yml

```diff
@@ -0,0 +1,32 @@
+name: Gource Action
+on:
+  push:
+    branches: [ master ]
+
+jobs: 
+  action:
+    timeout-minutes: 8
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: 'Checkout'
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    
+    - name: 'Gource Action'
+      uses: nbprojekt/gource-action@v1
+      with:
+        gource_title: 'Peirce My Heart'
+        logo_url: 'https://i.pinimg.com/736x/4e/35/fe/4e35fe64dbcd102d48e1fef43613d6e7.jpg'
+        gource_resolution: '1080p'
+        gource_fps: 60
```

Comment:
Id prefer to keep these video artifacts small, 24fps (used for TV) with 720p should probably be resolution sufficient.